mobify/platform-scaffold
# [Integration Manager] Checkout Schema Part 1

This PR contains the first half of the checkout schema work. It has become large enough that it could use to be split up into smaller PRs.

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-69
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Add types for the full checkout schema
- Convert the location logic
- Convert the shipping methods
- Switch to the new recommended method of connecting the Redux dev tools
- Clean up connector code in preparation for schematization.

## How to test-drive this PR
- Preview the checkout in Merlins and DW
- Ensure that it is still completable as far as is implemented in the base branch.
